testing/ncids-js-testing/public/index.html

```diff
@@ -1,48 +1,38 @@
 <html lang="en">
 	<head>
 		<title>Example Inner Page</title>
-		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
 	</head>
 	<body>
 		<a class="usa-skipnav" href="#main-content">Skip to main content</a>
-		<section class="usa-banner usa-banner--nci-banner" aria-label="Official government website">
+		<section
+			class="usa-banner usa-banner--nci-banner"
+			aria-label="Official government website"
+		>
 			<header class="usa-banner__header">
 				<div class="usa-banner__inner">
 					<div class="usa-banner__header-text">
 						An official website of the United States government
 					</div>
-					<button href="/" class="usa-button usa-button--nci-small">Español</button>
+					<button href="/" class="usa-button usa-button--nci-small">
+						Español
+					</button>
 				</div>
 			</header>
 		</section>
 
 		<div class="notifications-area">
 			<section
-				aria-label="Slim emergency site alert example"
-				class="usa-site-alert usa-site-alert--nci-slim usa-site-alert--nci-emergency"
-				id="site-alert-example1"
-			>
-				<div class="usa-alert">
-					<div class="usa-alert__body">
-						<div class="usa-alert__text">
-							<strong>Short alert message.</strong> Additional context and followup
-							information including <a class="usa-link" href="#">a link</a>.
-						</div>
-					</div>
-				</div>
-			</section>
-			<section
-				aria-label="Standard information site alert example with close"
+				aria-label="Standard information site alert example"
 				class="usa-site-alert usa-site-alert--nci-standard usa-site-alert--nci-info"
-				id="site-alert-example6"
-				data-site-alert-closable="true"
+				id="site-alert--nci-info"
 			>
 				<div class="usa-alert">
 					<div class="usa-alert__body">
-						<div class="usa-alert__nci-header">
+						<header class="usa-alert__nci-header">
```

Comment:
why was this updated here? if this is a bug, we should make sure this is updated everywhere, not just the example site

Reply:
reverted